packages/Webkul/Core/src/Core.php

```diff
@@ -605,21 +605,81 @@ public function formatPrice($price, $currencyCode = null)
 
         $formatter->setAttribute(\NumberFormatter::FRACTION_DIGITS, $currency->decimal ?? 2);
 
-        if (! $currency) {
+        if (!$currency) {
             return $formatter->formatCurrency($price, $currencyCode);
         }
 
         if ($symbol = $currency->symbol) {
             if ($this->currencySymbol($currency) == $symbol) {
-                return $formatter->formatCurrency($price, $currency->code);
-            }
 
+                return $this->formatPriceWithSymbol($price, $currency, $currency->symbol, $formatter);
+                // _________________________________________
+
+                // return $formatter->formatCurrency($price, $currency->code);
+            }
             $formatter->setSymbol(\NumberFormatter::CURRENCY_SYMBOL, $symbol);
 
-            return $formatter->format($this->convertPrice($price));
+            return $this->formatPriceWithSymbol($price, $currency, $currency->symbol, $formatter);
+            // _______________________
+            // $formatter->format($this->convertPrice($price));
+
+        }
+        return $this->formatPriceWithSymbol($price, $currency, $currency->code, $formatter);
+        // _______________________________________
+        // return $formatter->formatCurrency($price, $currency->code);
+    }
+
+    /**
+     *
```

Comment:
Doc comment should start from here, no extra line should be in the start. And after one info there should be one more extra line.

Reply:
Done.